04 - Object Methods/string_interpolation.rb

```diff
@@ -0,0 +1,31 @@
+name="Haris Butt"
+puts "hello my name is #{name}"
+
+puts "my " + name + " is "
+
+age=20
+puts "my age is #{age}"
+puts "in 10 years i will be #{age + 10}"
+
```

Comment:
It would be clearer if the comments and code were structured together for example.
```
=begin
3. Date and Time Formatting
Using Ruby’s Time.now, display the current date and time in the format: "Today is [weekday], [month] [day], [year] at [hour]:[minute]:[second]."
=end

puts "Today is #{Time.now.strftime('%A')}, #{Time.now.strftime('%B')} #{Time.now.strftime('%d')}, #{Time.now.strftime('%Y')} at #{Time.now.strftime('%H:%M:%S')}."
Footer
```